ga-ccd-definition/CaseEvent/User/UserEventsGAspec-JO-nonprod.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "INITIATE_COSC_APPLICATION_AFTER_PAYMENT",
+    "Name": "Cosc Applic After Payment",
```

Comment:
Cosc Application After Payment - would be better instead of applic

Reply:
i think it might hit the character limit?

Reply:
actually that's bang on 30 characters! will push the changes